fearless_simd/examples/play.rs

```diff
@@ -1,6 +1,11 @@
 // Copyright 2024 the Fearless_SIMD Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+#![expect(
+    missing_docs,
```

Comment:
FYI, for examples, `missing_docs` "should" only be complaining about the crate level docs being missing (i.e. a short explanation of the example). That's not to say those should be added in this PR, but more to make fixing these feel more tractable.